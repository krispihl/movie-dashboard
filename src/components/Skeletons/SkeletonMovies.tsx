import { Shimmer } from "./Shimmer";
import styles from './index.module.scss';

export const SkeletonMovies = () => {
    return (
        <div className={styles.skeletonMovies}>
            {Array.from(Array(100), () => 0)
                .map((_, i) => {
                    return (
                        <div className={styles.skeletonMovie} key={i}>
                            <Shimmer />
                        </div>
                    )
                })}
        </div>
    )
};
