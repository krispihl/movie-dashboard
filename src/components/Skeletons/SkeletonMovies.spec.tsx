import { SkeletonMovies } from "./SkeletonMovies";
import { render } from '@testing-library/react';

describe('src/components/SkeletonMovies', () => {
    it('should render SkeletonMovies correctly', () => {
        const { container } = render(<SkeletonMovies />);

        expect(container).toMatchSnapshot();
    })
});
