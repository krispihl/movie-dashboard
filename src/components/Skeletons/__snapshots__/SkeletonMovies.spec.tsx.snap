// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/SkeletonMovies should render SkeletonMovies correctly 1`] = `
<div>
  <div
    class="skeletonMovies"
  >
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
    <div
      class="skeletonMovie"
    >
      <div
        class="shimmerWrapper"
      >
        <div
          class="shimmer"
        />
      </div>
    </div>
  </div>
</div>
`;
